apps/nextjs-website/react-components/components/PreFooter/PreFooter.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, Stack, Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+import Image from 'next/image';
+import { BackgroundColor } from '../common/Common.helpers';
+import { isJSX } from '../../types/common/Common.types';
+import { PreFooterProps } from '@react-components/types/PreFooter/PreFooter';
+import { useTheme, useMediaQuery } from '@mui/material';
+import googleBadgeLightBase64 from './BadgeImages/googleBadgeLightBase64';
+import appleBadgeLightBase64 from './BadgeImages/appleBadgeLightBase64';
+import googleBadgeDarkBase64 from './BadgeImages/googleBadgeDarkBase64';
+import appleBadgeDarkBase64 from './BadgeImages/appleBadgeDarkBase64';
+import ContainerRC from '../common/ContainerRC';
+
+                
+const styles = {
+  main: (isSmallScreen: boolean) => ({
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: isSmallScreen ? 'column' : 'row',
+    padding: { md: '64px 24px', xs: '32px 24px' },
+    position: 'relative',
+    zIndex: 1,
+  }),
+  backgroundImage: (isSmallScreen: boolean, theme: 'light' | 'dark') => ({
+    backgroundColor: theme === 'dark' ? '#031344' : BackgroundColor(theme),
+    backgroundSize: isSmallScreen ? 'cover' : '30%',
+    backgroundPosition: isSmallScreen ? 'center' : 'right',
+    backgroundRepeat: 'no-repeat',
+    right: 0,
+  }),
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'relative',
+  },
+};
+
+const PreFooter = (props: PreFooterProps) => {
+  const { theme, title, storeButtons, background } = props;
+  const muiTheme = useTheme();
+  const isSmallScreen = useMediaQuery(muiTheme.breakpoints.down('sm'));
+
+  const googleBadgeBase64 =
+    theme === 'dark' ? googleBadgeLightBase64 : googleBadgeDarkBase64;
+  const appleBadgeBase64 =
+    theme === 'dark' ? appleBadgeLightBase64 : appleBadgeDarkBase64;
+
+  const BackgroundImage = isJSX(background) ? (
```

Comment:
The `isJSX` check is useless.

Background is always going to be either `undefined` or a string containing the background's URL.

Reply:
I'd just move the all `Box` component down to line 75 after a check to see if background is undefined.